Components/badge/badge.css

```diff
@@ -0,0 +1,77 @@
+@import url(../avatar/avatar.css);
+:root {
+  --gu-green: #008000;
+  --gu-red: #ff0000;
+  --gu-white: #ffffff;
+  --gu-grey: #808080;
+  --gu-l-green: #55dd33;
+}
+.gu-badge {
+  position: relative;
+  display: inline-block;
+  margin: 0.5rem;
+}
+.online {
+  width: 15px;
+  height: 15px;
+  background: var(--gu-green);
+  border-radius: 50%;
+  position: absolute;
+  right: 0;
+  bottom: 2px;
+  border: 2px solid var(--gu-white);
+}
+.dnd {
```

Comment:
what signifies dnd make it readable for others one suggestion to add comments atleast